docs/docs-new/pages/product/configuration/data-sources.mdx

```diff
@@ -78,6 +78,11 @@ can also contribute a driver, see this
     imageUrl="https://static.cube.dev/icons/trino.svg"
     title="Trino"
   />
+  <GridItem
+    url="data-sources/duckdb"
+    imageUrl="https://static.cube.dev/icons/duckdb-light.svg"
+    title="DuckDB / MotherDuck"
+  />
```

Comment:
The alignment seems to be a bit off. @hassankhan Wish your CSS-fu works here 😄 

<img width="903" alt="Screenshot 2023-08-11 at 18 21 08" src="https://github.com/cube-js/cube/assets/3852894/8a1c33f6-ccb5-4eb7-b66d-e40acdb971d7">